bindata/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -73,6 +89,9 @@ spec:
           }
           trap deletePid EXIT
 
+          chown -R openvswitch:openvswitch /var/run/openvswitch
+          chown -R openvswitch:openvswitch /etc/openvswitch
```

Comment:
If we are using systemd's ovsdb server shouldn't we use systemd's vswitchd as well?

Reply:
we do.